### PR TITLE
update skill tutorial for length parameter in table action type

### PR DESCRIPTION
**Pull request by DravitLochan**
Fixes issue #203

### CODE REVIEW

**Comment by DravitLochan** [comment]
@dynamitechetan @saurabhjn76 @chashmeetsingh @betterclever @Manveer695 @AliAyub007 @mayank408 @chiragw15 Please review!

File: README_susi_skill_language_tutorial.md [hunk 1]
```diff
@@ -387,7 +387,25 @@ The client then should create a table out of the data object where the column na
...
+Now let us say that if a skill developer wants only top 5 rows to be displayed at a time (as at times, APIs send large number JSONArray
+encoded responses.). For that we have an attribute "length" in table action type. Now if you want only top n rows should be displayed, 
+then modify the skill accordingly : 
+
+```
+...
+  "actions":[{
+    "type":"table",
+    "columns":{"currency":"Valuta","rate":"Quota"},
+    "length":"n"
```
Comment (chashmeetsingh) [inline README_susi_skill_language_tutorial.md:400]: we pass it as a string? Like inside quotes? or as a number?

Comment (DravitLochan) [reply README_susi_skill_language_tutorial.md:400]: As a string. for example look at http://dream.susi.ai/p/universities.

Comment (chashmeetsingh) [reply README_susi_skill_language_tutorial.md:400]: ok. please squash the commits.

Comment (DravitLochan) [reply README_susi_skill_language_tutorial.md:400]: Done. Please approve now!

**Review by AliAyub007 — APPROVED** [review]

**Review by chashmeetsingh — APPROVED** [review]

**Review by saurabhjn76 — APPROVED** [review]

**Review by madhavrathi — APPROVED** [review]

**Review by dynamitechetan — APPROVED** [review]

**Review by Laughing-Boy — APPROVED** [review]

**Pull request merged**